index/scorch/scorch.go

```diff
@@ -387,13 +387,23 @@ func (s *Scorch) Batch(batch *index.Batch) (err error) {
 	analysisResults := make([]index.Document, int(numUpdates))
 	var itemsDeQueued uint64
 	var totalAnalysisSize int
+	analysisBytes := func(tokMap index.TokenFrequencies) (rv uint64) {
```

Comment:
Should we add a flag here to calculate this only when the flag is set? 

Feel like this is something we don't always necessarily need to do - hope none of this calculation shows up in CPU profiles?

Reply:
Not sure if this is enough, we've to record copies - stored values and doc values. Also term vectors. I don't see you doing this in your zap PR.

Reply:
Just to close the thread, these two changes are going to be included in upcoming PRs.